assets/rollup.config.js

```diff
@@ -12,7 +12,7 @@ const json = require('@rollup/plugin-json');
 const resolve = require('@rollup/plugin-node-resolve');
 const replace = require('@rollup/plugin-replace');
 const {terser} = require('rollup-plugin-terser');
-
+const bubleConfig = require('./buble.config');
```

Comment:
I also saw that. Interestingly for Jest we need buble or babel (Jest uses babel by default) to transpile because they use commonJs modules (using require). Their support for ES modules is experimental. So I decided to keep it for now. But for sure something we need to keep an eye on. I think once Jest has real ES module support we should consider it.